docs/generated/errors-list.md

```diff
@@ -130,8 +130,15 @@ The `galasactl` tool can generate the following errors:
 - GAL1134E: The runs reset operation failed. Unable to process the error information returned from the server.
 - GAL1135E: Error cancelling run '{}'. Reason: '{}'
 - GAL1136E: The runs cancel operation failed. Unable to process the error information returned from the server.
+<<<<<<< HEAD
+- GAL1137E: Unable to compile the regex pattern for Galasa Property field 'namespace'. Reason: '{}'
```

Comment:
Can you please update the errors to prevent the conflict. running a build locally should fix this